studio-docs/source/org/members.mdx

```diff
@@ -7,36 +7,36 @@ A Studio organization can have any number of members, and each member can be ass
 >
 > If you're using member roles, [please let us know what you think!](https://docs.google.com/forms/d/10C4_ETX5Xb60RfnJjBC2xBVK9gjr6A644AYKA-27T-E/viewform) Your feedback will help us improve the feature before we roll it out to all organizations.
 
 Each organization member has one of the following roles:
 
 | Name                 | Description                                                                                                                                                                                                        |
 | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `Organization Admin` | Has management access to the entire organization, including its members, graphs, and configuration.                                                                                                                |
+| `Org Admin` | Has management access to the entire organization, including its members, graphs, and configuration.                                                                                                                |
 | `Graph Admin`        | Has management access to the organization's graphs, but not to members or organization-level configuration. Intended for developers who are considered admins to the graph.                                        |
 | `Contributor`        | Can push new schemas to graphs, but cannot configure graph settings or integrations. Intended for back-end developers who are authorized to make changes to graphs. _Cannot_ push schema changes to [protected variants](./graphs/#protected-variants-enterprise-only).                                               |
 | `Observer`           | Has view-only access to the organization's graph data, such as schemas and metrics. Can also execute queries in the Explorer. Intended for back-end developers who are _not_ authorized to make changes to graphs. |
-| `Consumer`           | Has view-only access to the organization's graph _schemas_, but not metrics. Can also execute queries in the Explorer. Intended for application developers building clients against a graph.                       |
-| `Billing Manager`    | Has management access to organization-level configuration and billing. Can also remove members (but not invite them). No access to graphs.                                                                         |
+| `Consumer` (free)        | <p>Has view-only access to the organization's graph _schemas_, but not metrics. Can also execute queries in the Explorer. Intended for application developers building clients against a graph.</p><p>This role is available to all plan types, including Free. If you have a paid plan, `Consumer` members do not count as billed users.</p>                    |
```

Comment:
We used the term "free forever" in the app, maybe we'd want to use that here too?

Reply:
I think because we'll be adding "free" to the Billing Manager row just below, I'll keep this as just "free" for clarity